c++ training/W2S1/E4/Source.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+
+//Create a printer class.This class should allow you to store a string and then output it in different formats.Ask the user to input a string and then show what your printer can do!Remember to use char arrays instead of stringsand avoid the std namespace.
+//
+//You class should have the following methods :
+//Printer();
+//Printer(std::string outputString);
+//void Print();
+//void PrintReversed();
+//void PrintSpaced(int spaces);
+//void PrintVertically();
+
+
+class Printer {
+private:
+	char m_outputString[60];
+	void Filler(const char newString[]) {
```

Comment:
Remember that methods should start with verbs to communicate an action. Filler is a noun and is more suitable for an object, rather than a function.